src/web/app/components/question-submission-form/question-submission-form.component.ts

```diff
@@ -98,6 +98,41 @@ export class QuestionSubmissionFormComponent implements OnInit {
   }
 
   ngOnInit(): void {
+    this.sortRecepientsByName();
+  }
+
+  /**
+   * Sorts recepients of feedback by their name.
+   *
+   * Feedback recepients are initially sorted by their email address.
+   */
+  private sortRecepientsByName(): void {
+    this.model.recipientList.sort((firstRecepient: FeedbackResponseRecipient,
+      secondRecepient: FeedbackResponseRecipient) => {
+      const firstRecepientNameLowerCase: string = firstRecepient.recipientName;
+      const secondRecepientNameLowerCase: string = secondRecepient.recipientName;
+
+      if (firstRecepientNameLowerCase < secondRecepientNameLowerCase) {
+        return -1;
+      }
+
+      if (firstRecepientNameLowerCase > secondRecepientNameLowerCase) {
+        return 1;
+      }
+
+      return 0;
```

Comment:
this block can be replaced by `firstRecepient.recipientName.localeCompare(secondRecepient.recipientName)`

Reply:
Changed it as suggested!